server/main.lua

```diff
@@ -1,5 +1,17 @@
 local config = require 'config.server'
 
+---@param vehNetId number
+---@param state number
+local function setLockState(vehNetId, state)
+    local vehicleEntity = NetworkGetEntityFromNetworkId(vehNetId)
+	if type(state) ~= 'number' or not DoesEntityExist(vehicleEntity) then return end
+    local vehicleConfig = GetVehicleConfig(vehicleEntity)
+    if vehicleConfig.noLock or vehicleConfig.shared then return end
+    Entity(vehicleEntity).state:set('doorslockstate', state, true)
+end
+
+exports('SetVehLockState', setLockState)
```

Comment:
I think we should call this SetLockState since qbx_vehicles implies the vehicle part already